bubblewrap.c

```diff
@@ -86,6 +86,7 @@ static bool opt_unshare_cgroup_try = FALSE;
 static bool opt_needs_devpts = FALSE;
 static bool opt_new_session = FALSE;
 static bool opt_die_with_parent = FALSE;
+static bool opt_signal_propogate = FALSE;
```

Comment:
s/propogate/propagate/

Reply:
Yes, or rename it to something easier to spell like `--forward-signals`.